arrow/obsidianmodule.c

```diff
@@ -0,0 +1,468 @@
+#include <Python.h>
```

Comment:
This file looks nightmarish.  Is there a good reference for developing this sort of Python-NumPy-C code?

Reply:
To @jmason42 's question about reference docs: also https://docs.python.org/2/c-api/index.html